packages/libspa-codec-bluez5-aac-bin/libspa-codec-bluez5-aac-bin.pacscript

```diff
@@ -0,0 +1,18 @@
+name="libspa-codec-bluez5-aac-bin"
+gives="libspa-codec-bluez5-aac"
+version="0.3.65-4~glasgall1"
+url="https://github.com/Zahrun/libspa-0.2-bluetooth/raw/main/libspa-0.2-bluetooth_${version}_amd64.deb."
```

Comment:
```suggestion
url="https://github.com/Zahrun/libspa-0.2-bluetooth/raw/main/libspa-0.2-bluetooth_${version}_amd64.deb"
```

Reply:
If the extension is .deb, pacstall will install the deb file directly. The additionnal dot is intentional.

Reply:
Why can't the deb just be installed again?

Reply:
It depends on stuff that are in the ppa libspa-0.2-modules (= 0.3.65-4~glasgall1) but 0.3.71-0~tux1 is to be installed
or in specific ubuntu versions libsbc1 (>= 2.0) but 1.5-3build2 is to be installed
Whereas just adding the one missing .so file at the right spot works just fine

Reply:
So can you not create a `tar.xz` with just the `.so` file in it?

Reply:
I could, but then I have to do that every time there is a new version, whereas here I am just taking it out from the deb.
I’m fine to do the tar.xz if you prefer that.

Reply:
I'd like a `.tar.xz`.

Reply:
Ok, although less automatic, this looks cleaner, I must admit